scripts/production/batch_lastz.pl

```diff
@@ -184,7 +184,7 @@
 }
 # Create all JIRA tickets
 my $subtask_keys = $jira_adaptor->create_tickets(
-    -JSON_INPUT => encode_json($ticket_list), -ISSUE_TYPE => 'Sub-task', -DRY_MODE => $dry_mode);
+    -JSON_INPUT => encode_json($ticket_list), -ISSUE_TYPE => 'Sub-task', -DRY_MODE => $dry_run);
```

Comment:
Let's also change the `create_tickets` option to be named `-DRY_RUN`, just for consistency?